DefaultTabBar.js

```diff
@@ -45,7 +45,7 @@ const DefaultTabBar = React.createClass({
       accessibilityTraits='button'
       onPress={() => this.props.goToPage(page)}
     >
-      {this.renderTabName(name, page, isTabActive)}
+      {this.props.renderTabName(name, page, isTabActive)}
```

Comment:
```renderTabName``` I'm thinking that it would be more flexible to make ```renderTab``` method. For example if someone wants ```TouchableHighlight```.